lib/active_merchant/billing/gateways/decidir_plus.rb

```diff
@@ -321,8 +321,15 @@ def error_message(response)
         return error_code_from(response) unless validation_errors = response.dig('validation_errors')
 
         validation_errors = validation_errors[0]
+        message = "#{validation_errors&.dig('code')}: #{validation_errors&.dig('param')}"
+        message == ': ' ? nil : message
```

Comment:
Do we need lines 323-325 since on line 286 we are now calling `validation_error_message`. Or is error message called somewhere else as well?

Reply:
I could update error_message do be like validation_error_message. That would make a lot more sense

Reply:
I didn't want to change how we format the current error message because that will probably need customer comm.